src/sentry/notifications/notifications/integration_nudge.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+
+import logging
+import random
+from typing import TYPE_CHECKING, Any, Iterable, Mapping, MutableMapping, Sequence
+
+from sentry.db.models import Model
+from sentry.notifications.notifications.base import OrganizationNotification
+from sentry.notifications.utils import send_base_notification
+from sentry.notifications.utils.actions import MessageAction
+from sentry.types.integrations import ExternalProviders
+
+if TYPE_CHECKING:
+    from sentry.models import Organization, Team, User
+
+logger = logging.getLogger(__name__)
+
+MESSAGE_LIBRARY = [
```

Comment:
Is there any way for us to know which message(s) was more effective than another?

Reply:
My plan was to just run the experiments far enough away from each other that we can just see when the numbers go up the fastest.

Reply:
@mgaeta I would put this info in the analytics event